assessment.sql

```diff
@@ -5,24 +5,56 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees(
+  id SERIAL PRIMARY KEY,
+  surname CHARACTER VARYING,
+  given_name CHARACTER VARYING,
+  genderCHARACTER VARYING(1),
+  height INTEGER,
+  weight INTEGER,
+  dob DATE,
+  eye_color CHARACTER VARYING,
+  hair_color CHARACTER VARYING)
+;
 
 -- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
-
--- your code here
+INSERT INTO licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color)
+VALUES('Bishop', 'Caren', 'f', 63, 132, '1943-09-26', 'Brown', 'Black');
 
 -- Question 3. Load the data from `data/licensees.csv` into the `licensees` table.
 
--- your code here
+--assuming we're using a bulk_load script to do this action. Purposely setting the first line of the csv file as the table header
+\copy licensees(surname,given_name,gender,height,weight,dob,eye_color,hair_color) FROM 'data/licensees.csv' WITH (HEADER true, FORMAT csv)
 
 -- Question 4. Write a query to get all attributes of licensees with Hazel eye color and Brown or Black hair color.
 
--- your code here
+
+
+-- not sure of syntax for selecting multiple columns with varying values
+SELECT *
+FROM licensees
+WHERE eye_color = 'Hazel' AND hair_color = 'Brown'
+  OR hair_color = 'Black'
+;
```

Comment:
You're in for a surprise if you ever run this statement against a suitable table. Rather than doing what the question asked for, it will search for rows where both `eye_color` is `'Hazel'` and `hair_color` is `'Brown'`, or just `hair_color` is `'Black'`.

This is because AND has precedence over OR. In order to get what you want, you must use parentheses to force a certain order of operations. 4/5